test.disabled/ejabberd_tests/rebar.config

```diff
@@ -7,7 +7,7 @@
         {escalus, ".*", {git, "git://github.com/esl/escalus.git", "e1eca08218c21f1ac9d45431194e4463119efff6"}},
         {exml, ".*", {git, "git://github.com/esl/exml.git", "2.4.0"}},
```

Comment:
Could we change exml version to newer? 2.4.0 is giving compilation errors if certain flags are set in environment, newer versions are safe.